.ci.yaml

```diff
@@ -1399,6 +1399,16 @@ targets:
       task_name: android_obfuscate_test
     scheduler: luci
 
+  - name: Linux_android android_semantics_integration_test
+    recipe: devicelab/devicelab_drone
```

Comment:
Maybe `bringup: true` (in the bringup and not flaky sense) in case there's something different in prod, so we can see it pass at least a few times?

Reply:
The env in staging an prod should be the same, but just in case, added the `bringup: true`. Will leave it to the `auto flake bot` to deflake.